torchvision/csrc/ops_autoregister.h

```diff
@@ -0,0 +1,17 @@
+#include "ops.h"
+
+#if (defined __cpp_inline_variables) || __cplusplus >= 201703L
+#define VISION_INLINE_VARIABLE inline
+#else
+#ifdef _MSC_VER
+#define VISION_INLINE_VARIABLE __declspec(selectany)
+#else
+#define VISION_INLINE_VARIABLE __attribute__((weak))
+#endif
+#endif
+
+namespace vision {
+namespace impl {
+VISION_INLINE_VARIABLE int dummy = RegisterOps();
```

Comment:
So the key problem is why this line is not working, right?

Reply:
I suggest removing the try..catch block in https://github.com/pytorch/vision/blob/master/torchvision/extension.py#L23-L27 to see whether there are skipped errors while loading the extension library.

Reply:
I get an interesting finding here. When I use the Debug configuration, it works. But for Release, it doesn't. So it seems that the variable is optimized away.

Reply:
Adding `__declspec(dllexport)` should force the linker not to optimize it away. See https://github.com/pytorch/vision/pull/2489/files#diff-92e9c21d561ea7205d212a64badbc082R13-R17.